src/SarifWeb/wwwroot/css/Validator.css

```diff
@@ -31,6 +31,10 @@
     font-size: 20px;
 }
 
+#validatorToolbar *[no-disable] {
```

Comment:
>#validatorToolbar *[no-disable] [](http://example.com/codeflow?start=0&length=31)

is this the exact opposite and overwrite of:
#validatorToolbar[data-disabled]
just thinking about the name if we want similar name
#validatorToolbar[no-data-disabled]
but I think no-disable is clear.